### PR TITLE
chanreg: Explain how to use benefits of a community

### PR DESCRIPTION
**Pull request by ProgVal**

### CODE REVIEW

File: chanreg.md [hunk 1]
```diff
@@ -205,9 +205,10 @@ will then get in touch with you.
 
 Registered communities can profit from the following benefits:
 
-- Channels in their namespace can be claimed
+- Channels in their namespace can be claimed (with `/msg ChanServ CLAIM`)
 - Optional cloaks are available for members.
-  These replace the hostname part with `@about/yourcommunity/username`
+  These replace the hostname part with `@about/yourcommunity/username`,
```
Comment (Fuchs) [inline chanreg.md:210]: Format will potentially change to 
@about/yourcommunity/foo/bar   where foo is optional, same as with project cloaks. 
Once we decided on that (currently on the agenda for the next meeting) it makes sense to document it for both projects and communities.

Comment (ProgVal) [reply chanreg.md:210]: Isn't this out of scope for this PR? It's not about the format

Comment (Fuchs) [reply chanreg.md:210]: I'd wait until we decided it  (currently planned: ~ june 11th) so we don't have to adopt it yet again, but that's a soft block from me, if people think it's fine to already document it in the meantime, that's fine with me

Comment (swantzter) [reply chanreg.md:210]: @Fuchs it's already documented (see https://libera.chat/chanreg#benefits-of-a-community) the only change on this line is the addition of a comma

So I think this is fine to push, and we can do any other changes caused by meetings separately

Comment (Fuchs) [reply chanreg.md:210]: Oh, oops, sorry, yes, all fine :)

**Pull request merged**